Sources/XCTest/XCTestMain.swift

```diff
@@ -88,64 +63,30 @@ internal struct XCTRun {
     //   "All tests".
     // - An `XCTestSuite` representing the .xctest test bundle is not included.
     let selectedTestName = ArgumentParser().selectedTestName
-    var rootTestSuites = [XCTestSuite]()
+    let rootTestSuite: XCTestSuite
+    let currentTestSuite: XCTestSuite
     if selectedTestName == nil {
-        rootTestSuites.append(XCTestSuite(name: "All tests"))
-        rootTestSuites.append(XCTestSuite(name: "\(testBundle.bundlePath.lastPathComponent).xctest"))
+        rootTestSuite = XCTestSuite(name: "All tests")
+        currentTestSuite = XCTestSuite(name: "\(testBundle.bundlePath.lastPathComponent).xctest")
```

Comment:
I wonder if it's a little misleading to append `.xctest` to the output here even though we're not actually executing a test bundle? I'm not sure how I feel about this just yet. 🤔